src/LiveComponent/composer.json

```diff
@@ -29,7 +29,9 @@
         "php": ">=8.1",
         "symfony/deprecation-contracts": "^2.5|^3.0",
         "symfony/property-access": "^5.4.5|^6.0|^7.0",
+        "symfony/property-info": "^5.4|^6.0|^7.0",
```

Comment:
This package is a hard dependency, so it has to be moved to the `require` section.